addons/web_editor/models/ir_attachment.py

```diff
@@ -60,3 +62,53 @@ def _get_media_info(self):
         """Return a dict with the values that we need on the media dialog."""
         self.ensure_one()
         return self._read_format(['id', 'name', 'description', 'mimetype', 'checksum', 'url', 'type', 'res_id', 'res_model', 'public', 'access_token', 'image_src', 'image_width', 'image_height', 'original_id'])[0]
+
+    def _get_html_fields(self):
+        return [('ir.ui.view', 'arch_db', [('type', '=', 'qweb')])]
+
+    def _is_used_in_html_field(self):
+        """Returns a model where this attachment is used, or a dict with model names or False."""
+        self.ensure_one()
+        # Keep significant part of attachment url
```

Comment:
I actually thought you would already consider the 16.0/master change/imp and make it so this method can be called on any model.
I don't remember exactly where we need it but at least for page dependencies (to adapt the imp I made which is not sudo/active compliant) but also for the property fields in forms where we discussed today that we might just prevent the deletion of a field if it's used in the website.

Reply:
Oh, you mean defining it on `base`'s extension in `web_editor` ? Ok.